system-int-test/test/cmr/system_int_test/search/collection_consortium_geoss_search_test.clj

```diff
@@ -0,0 +1,78 @@
+  (ingest/create-provider {:provider-guid "provguid_geoss_fod_false" :provider-id "PROV4" :consortiums "geoss cstg41 cstg42"})
+  (ingest/create-provider {:provider-guid "provguid_no_geoss_fod_nil_with_geossurl" :provider-id "PROV5" :consortiums "cstg51 cstg52"})
+  (ingest/create-provider {:provider-guid "provguid_no_geoss_fod_nil_no_geosurl" :provider-id "PROV6" :consortiums "cstg61 cstg62"})
+
+  (let [;; no UseConstraint, eosdis provider: geoss should be added to the consortiums list.
+        coll1 (d/ingest-umm-spec-collection
+               "PROV1"
+               (umm-c/collection 1
+                                 {:CollectionCitations [{:Creator "K. Hilburn"}
+                                                        {:Creator "Nelkin"}]})
+               {:format :umm-json})
+        ;; no UseConstraint, not eosdis provider: no geoss should be added to the consortums list. 
+        coll2 (d/ingest-umm-spec-collection
+               "PROV2"
+               (umm-c/collection 1 
+                                 {:CollectionCitations [{:Creator "K. Hilburn, J. Ardizzone, and S. Gao"
+                                                         :OtherCitationDetails "Sounder PEATE (Product Evaluation and Test Element) Team/Ruth Monarrez, JPL"}]})
+               {:format :umm-json})
+
+        ;; UseConstraint with FreeAndOpenData being true, no GEOSS provider: geoss should be added to the consortiums list.
+        coll3 (d/ingest-umm-spec-collection
+               "PROV3"
+               (umm-c/collection 1
+                                 {:UseConstraints {:Description "PROV3" :FreeAndOpenData true}})
+               {:format :umm-json})
+
+        ;; UseConstraint with FreeAndOpenData being false, GEOSS provder: geoss shouldbe removed from the consortiums list.
+        coll4 (d/ingest-umm-spec-collection
+               "PROV4"
+               (umm-c/collection 1
+                                 {:UseConstraints {:Description "PROV4" :FreeAndOpenData false}})
+               {:format :umm-json})
+
+        ;; UseConstraint with FreeAndOpenData being nil, and GEOSS url: geoss should be added to the consortiums list.
+        coll5 (d/ingest-umm-spec-collection
+               "PROV5"
+               (umm-c/collection 1
+                                 {:UseConstraints {:Description "PRO5" :LicenseURL {:Linkage "http://creativecommons.org/licenses/by/4.0/"}}})
+               {:format :umm-json})
+
+        ;; UseConstraint with FreeAndOpenData being nil, no GEOSS url: no geoss should be added to the consortiums list.
+        coll6 (d/ingest-umm-spec-collection
+               "PROV6"
+               (umm-c/collection 1
+                                 {:UseConstraints {:Description "PRO6" :LicenseURL {:Linkage "http://not-geoss-url"}}})
+               {:format :umm-json})]
+
+    (index/wait-until-indexed)
+
+    (testing "consortium parameter geoss search"
+      (are3 [items consortium options]
```

Comment:
there is only one test case, no need to use `are3` here.

Reply:
If you look at the description Erich put in there. When there is use-constraints, EOSDIS provider or not doesn't play a role in it.